entity-management-web/src/main/java/eu/europeana/entitymanagement/exception/ingestion/EntityValidationException.java

```diff
@@ -22,8 +22,12 @@ public EntityValidationException(String message, Set<ConstraintViolation<Entity>
 
     @Override
     public String toString() {
-	StringBuilder message = new StringBuilder(super.toString());
-	return getConstrainsValidationMessage(message);
+	return buildErrorMessage();
+    }
+
+    String buildErrorMessage() {
+        StringBuilder message = new StringBuilder(super.toString());
```

Comment:
This line still causes a StackOverflowError.

Reply:
true, it is fixed now in another branch